### DISCUSSION TITLE
Installing on my M1 Max Mac Studio does not work

### DISCUSSION THREAD

**Discussion by DavidFosterLSP** [Xaymar/obs-stream-effects#1004] [Q&A] [answered]
I followed the guide you created to a T and it still is not installing on my M1 Max Mac. I tried both the stable 0.11 and 0.12 and neither work. This is a new machine so it has never been installed on here before. But just in case I used terminal and removed them and then tried again, still nothing. Not sure what the issue is. Thanks in advance! This is one of my favorite plugins!

**Comment by Xaymar** [comment, marked as answer]
v0.11.1 requires OBS Studio 27.2.4. 0.12 is currently not a supported version for any work.
